diff -uNr openocd-0.12.0/src/server/server.c openocd-0.12.0_patched/src/server/server.c
--- openocd-0.12.0/src/server/server.c	2023-10-20 09:36:56.933799800 +0200
+++ openocd-0.12.0_patched/src/server/server.c	2023-10-20 09:40:27.510038377 +0200
@@ -102,7 +102,7 @@
 
 #ifdef _WIN32
 		/* we are using stdin/out so ignore ctrl-c under windoze */
-		SetConsoleCtrlHandler(NULL, TRUE);
+//		SetConsoleCtrlHandler(NULL, TRUE);
 #endif
 
 		/* do not check for new connections again on stdin */
@@ -598,6 +598,7 @@
 	return shutdown_openocd == SHUTDOWN_WITH_ERROR_CODE ? ERROR_FAIL : ERROR_OK;
 }
 
+#if 0
 static void sig_handler(int sig)
 {
 	/* store only first signal that hits us */
@@ -627,6 +628,7 @@
 		LOG_DEBUG("Ignored Signal %d", sig);
 }
 #endif
+#endif
 
 
 int server_host_os_entry(void)
@@ -661,18 +663,18 @@
 {
 #ifdef _WIN32
 	/* register ctrl-c handler */
-	SetConsoleCtrlHandler(control_handler, TRUE);
+//	SetConsoleCtrlHandler(control_handler, TRUE);
 
-	signal(SIGBREAK, sig_handler);
-	signal(SIGINT, sig_handler);
+//	signal(SIGBREAK, sig_handler);
+//	signal(SIGINT, sig_handler);
 #else
-	signal(SIGHUP, sig_handler);
-	signal(SIGPIPE, sig_handler);
-	signal(SIGQUIT, sigkey_handler);
-	signal(SIGINT, sigkey_handler);
+//	signal(SIGHUP, sig_handler);
+//	signal(SIGPIPE, sig_handler);
+//	signal(SIGQUIT, sigkey_handler);
+//	signal(SIGINT, sigkey_handler);
 #endif
-	signal(SIGTERM, sig_handler);
-	signal(SIGABRT, sig_handler);
+//	signal(SIGTERM, sig_handler);
+//	signal(SIGABRT, sig_handler);
 
 	return ERROR_OK;
 }
@@ -700,7 +702,7 @@
 	target_quit();
 
 #ifdef _WIN32
-	SetConsoleCtrlHandler(control_handler, FALSE);
+//	SetConsoleCtrlHandler(control_handler, FALSE);
 
 	return ERROR_OK;
 #endif
